### PR TITLE
Migrate react-bootstrap Button component in arena to MaterialUI Issue #177

### PR DESCRIPTION
**Pull request by annovo**
Pull-Request for `paretOS`

## Description
- Changed Button to material UI component in CreateSprintTemplate 
- Changed Button to IconButton in SprintCreation
- Changed Button to material UI Button and Chip in Sprint
- Added Button gradient variant and Chip variant to `libs/theme`

## Relates to
- [Refactor:  Migrate react-bootstrap Button component to MaterialUI](https://github.com/mikhael28/paretOS/issues/177)

## Reviewers
- @jayeclark 
- @mikhael28 

### Screenshots / other info
Screen Shot 2022-04-13 at 5 38 46 PM

Screen Shot 2022-04-13 at 5 53 22 PM

Screen Shot 2022-04-13 at 5 41 35 PM

Screen Shot 2022-04-13 at 2 01 53 PM
Screen Shot 2022-04-13 at 5 42 22 PM

### CODE REVIEW

File: src/arena/Sprint.js [hunk 1]
```diff
@@ -471,31 +483,41 @@ function Sprint(props) {
...
                         key={idx}
-                      >
-                        Day {idx + 1}
-                      </Button>
+                      />
                     ))}
                   </div>
                   <div className="flex">
-                    {props.user.admin === true ? (
+                    {props.user.admin === false ? (
```
Comment (mikhael28) [inline src/arena/Sprint.js:499]: Good morning @annovo - can you change this back to only being visible for `admin`?

File: src/arena/Sprint.js [hunk 2]
```diff
@@ -471,31 +483,41 @@ function Sprint(props) {
...
-                          <Button onClick={() => setStatus("early")}>
-                            Set to Early
-                          </Button>
-                          <Button onClick={() => setStatus("active")}>
-                            Set to Active
-                          </Button>
-                          <Button onClick={() => setStatus("inactive")}>
-                            Set to Inactive
-                          </Button>
+                        <div>
```
Comment (mikhael28) [inline src/arena/Sprint.js:502]: Great idea to use the `Chip` element! Better than using Buttons. Is the `div` surrounding the chips necessary? And the `div` surrounding the <p> tag and the Chips - can you use React.Fragments instead? They offer better performance, you can learn more here! https://www.sitepoint.com/react-fragments-introduction/

Comment (mikhael28) [reply src/arena/Sprint.js:502]: As you see random divs that don't seem to have associated styling with them, feel free to replace them with <React.Fragment> or <> elements so we can remove unnecessary divs from the repository!

File: src/libs/theme.js [hunk 3]
```diff
@@ -91,6 +91,55 @@ const theme = createTheme({
         },
       ],
     },
+    MuiButton: {
```
Comment (mikhael28) [inline src/libs/theme.js:94]: Great work!

**Review by mikhael28 — CHANGES_REQUESTED** [review]
Great work here! I have two small changes I would like you to make before merging this in - take a look at the PR and let me know if you have any questions!

File: src/arena/Sprint.js (hunk 1 above)
Comment (annovo) [reply src/arena/Sprint.js:499]: Sorry, I thought I did it with “fix admin setting” commit, but apparently when I resolved the conflict between branches I missed that it changed back :(

File: src/arena/Sprint.js (hunk 2 above)
Comment (annovo) [reply src/arena/Sprint.js:502]: Will do 🙂

Comment (mikhael28) [reply src/arena/Sprint.js:502]: Great! Just request 're-review' once you are done :) By the way, we have a contributor only Discord channel. We only have a few people, but if you would like to join - leave your email here and I will send you an invite.

Comment (annovo) [reply src/arena/Sprint.js]: I will be happy to join :) 

ananas.nasya@gmail.com

**Review by mikhael28 — APPROVED** [review]

**Comment by mikhael28** [comment]
Merged, great work! Will be sending you an invite to the Discord channel shortly!

**Pull request merged**